attributes/plugins.rb

```diff
@@ -69,7 +69,7 @@
   pipeline-utility-steps:1.4.0
   plain-credentials:1.4
   run-condition:1.0
-  scm-api:2.2.0
+  scm-api:2.2.6
```

Comment:
I don't see a security update for this plugin, but there isn't any major, breaking changes, so this is fine.

Reply:
I'm looking at https://plugins.jenkins.io/scm-api for the changes BTW

Reply:
I believe this was a dependency that was pulled in for one of the plugins that got updated.